ucan-invocation/functions/metrics-accumulated-size.js

```diff
@@ -0,0 +1,46 @@
+import * as Sentry from '@sentry/serverless'
+
+import { createW3MetricsTable } from '../tables/w3-metrics.js'
+import { parseKinesisEvent } from '../utils/parse-kinesis-event.js'
+import { STORE_ADD } from '../constants.js'
+
+Sentry.AWSLambda.init({
+  environment: process.env.SST_STAGE,
+  dsn: process.env.SENTRY_DSN,
+  tracesSampleRate: 1.0,
+})
+
+const AWS_REGION = process.env.AWS_REGION || 'us-west-2'
+
+/**
+ * @param {import('aws-lambda').KinesisStreamEvent} event
+ */
+async function handler(event) {
+  const ucanInvocations = parseKinesisEvent(event)
+
+  const {
+    TABLE_NAME: tableName = '',
+    // set for testing
+    DYNAMO_DB_ENDPOINT: dbEndpoint,
+  } = process.env
+
+  await updateAccumulatedSize(ucanInvocations, {
+    w3MetricsTable: createW3MetricsTable(AWS_REGION, tableName, {
+      endpoint: dbEndpoint
+    })
+  })
+}
+
+/**
+ * @param {import('../types').UcanInvocation[]} ucanInvocations
+ * @param {import('../types').W3AccumulatedSizeCtx} ctx
+ */
+export async function updateAccumulatedSize (ucanInvocations, ctx) {
+  const invocationsWithStoreAdd = ucanInvocations.filter(
+    inv => inv.value.att.find(a => a.can === STORE_ADD)
```

Comment:
Should we consider also `store/remove` into this calculation? Or, perhaps better - collect a separate metric that for `store/remove-accumulated-size` (followup PR)?

Reply:
yes, follow up PR with your suggestion. The goal of this PR is to get a "this is the way we should do things". Then follow up PRs with consumers for each metrics will be just follow same pattern

Reply:
Added in tracking https://github.com/web3-storage/w3infra/issues/117